source/install/verify-signatures/gpg.txt

```diff
@@ -0,0 +1,93 @@
+.. _mongosh-verify-signatures-gpg:
+
+==========================================
+Verify Packages with GPG (Linux and macOS)
+==========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. include:: /includes/verify-signatures-intro.rst
+
+This page describes how to use GPG to verify Linux and macOS packages.
+
+Before you Begin
+----------------
+
+.. include:: /includes/verify-signatures-before-you-begin.rst
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Import the MongoDB Shell public key
+
+      Run this command:
+
+      .. code-block:: sh
+
+         curl https://pgp.mongodb.com/mongosh.asc | gpg --import
+
+      .. include:: /includes/verification-gpg-results.rst
+
+   .. step:: Download the MongoDB Shell public signature
+
+      To download the MongoDB Shell public signature, go to the `mongosh
+      Releases <https://github.com/mongodb-js/mongosh/releases>`__ page
+      on GitHub and download the corresponding ``.sig`` file for your
+      version and variant.
+
+      For example, if you are running
+      ``mongodb-mongosh_{+version+}_amd64.deb``, download
+      ``mongodb-mongosh_{+version+}_amd64.deb.sig``
+
+      .. note:: 
+         
+         Make sure that you select the correct version in the GitHub
+         releases page when you download the signature.
+
+   .. step:: Verify the package
+
+      Run this command:
+
+      .. code-block:: sh
+
+         gpg --verify <path_to_signature_file> <path_to_mongosh_executable>
+
+      If the package is properly signed by MongoDB, the command returns
+      this response:
+      
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: Signature made Mon Jan 22 10:22:53 2024 CET
+         gpg:                using RSA key AB1B92FFBE0D3740425DAD16A8130EC3F9F5F923
+         gpg: Good signature from "Mongosh Release Signing Key <packaging@mongodb.com>" [unknown]
+         
+      If the package is properly signed but you do not trust the signing
```

Comment:
(jargon)
I might consider rewording to omit "you do not trust", which always sounds a little jargony to me. Maybe something like: If the page is properly signed but the signing key is not added to your local trustdb, gpg ...